lib/administrate/field/associative.rb

```diff
@@ -24,6 +24,10 @@ def associated_class_name
       def primary_key
         options.fetch(:primary_key, :id)
       end
+
+      def foreign_key
+        options.fetch(:foreign_key, nil)
```

Comment:
I'm not a huge fan of the way this reads in combination with line 25 above. Line 25 is handing you a default if not specified (which is what I would expect), but `foreign_key` is instead returning `nil`, and then allowing `permitted_attribute` to implement the default behavior for a `foreign_key`, but it's not clear from this line that that is what is going on. Does that make sense?